src/BodyPose/index.js

```diff
@@ -528,6 +528,17 @@ class BodyPose {
       );
     }
   }
+
+  /**
+   * Returns the skeletal connections pairs for the model.
+   * An alias for `getConnections`. This method is deprecated and will be removed in the future.
+   * @returns {number[][]} an array of pairs of indices containing the connected keypoints.
+   * @public
+   * @deprecated
```

Comment:
One last little detail here. Following up on my discussion with @gohai, would it make sense to take out "will be removed in the future" and well as "@deprecated" and just leave both methods in, one as an alias? Or is that confusing / bad practice? Would this make the documentation confusing to have two methods that do the same thing? cc @MOQN @QuinnHe @alanvww 

We can always change our mind later, we don't have to feel this is set in stone.

Reply:
I can take it out for now!